lib/improver/utilities/spatial.py

```diff
@@ -85,7 +85,7 @@ def check_if_grid_is_equal_area(cube):
 
 
 def convert_distance_into_number_of_grid_cells(
-        cube, distance, max_distance_in_grid_cells):
+        cube, distance, max_distance_in_grid_cells=10**4):
```

Comment:
I was wondering, rather than setting a default here for the `max_distance_in_grid_cells` to an arbitrarily large number, the default could be set to `None`, and the check on the `max_distance_in_grid_cells` would only take place, if this is set. For example:
```python
if max_distance_in_grid_cells:
    if (grid_cells_x > max_distance_in_grid_cells or
            grid_cells_y > max_distance_in_grid_cells):
```
This might be a bit more direct, as the aim here is to skip the check on `max_distance_in_grid_cells`.